CMakeLists.txt

```diff
@@ -79,11 +79,11 @@ if(HELFEM_FIND_DEPS)
 
         find_package(LibXc REQUIRED PATHS ".")
         message("LibXc_FOUND=${LibXc_FOUND}")
-        message("LIBXC_INCLUDE_DIRS=${LIBXC_INCLUDE_DIRS}")
-        message("LIBXC_LIBRARIES=${LIBXC_LIBRARIES}")
+        message("LIBXC_INCLUDE_DIRS=${Libxc_INCLUDE_DIRS}")
+        message("LIBXC_LIBRARIES=${Libxc_LIBRARIES}")
 
-        include_directories("${HDF5_INCLUDE_DIRS}" "${LIBXC_INCLUDE_DIRS}")
-        link_libraries("${HDF5_LIBRARIES}" "${LIBXC_LIBRARIES}")
+        include_directories("${HDF5_INCLUDE_DIRS}" "${Libxc_INCLUDE_DIRS}")
+        link_libraries("${HDF5_LIBRARIES}" "${Libxc_LIBRARIES}")
```

Comment:
This part was contributed by @mortenpi. On later examination, it looks like there is an issue with inconsistent capitalization. Libxc does have a working CMake system, as well. The usage should be something like
```
find_package(Libxc)
```
and the link should be against `Libxc::xc`

Reply:
Using `-DHELFEM_FIND_DEPS=ON`, I can confirm that `find_package(Libxc)` in `CMakeLists.txt` works fine with the Libxc-6.2.2 (currently available as stable bottle from Homebrew). Without the correct capitalization of `Libxc_INCLUDE_DIRS` and `Libxc_LIBRARIES`, `cmake` aborts:
```
LibXc_FOUND=1
LIBXC_INCLUDE_DIRS=
LIBXC_LIBRARIES=
CMake Error at CMakeLists.txt:85 (include_directories):
  include_directories given empty-string as include directory.
```